app.py

```diff
@@ -59,9 +67,10 @@ def str2bool(value: str) -> bool:
     stack_name=f"geospatial-data-lake-api-{ENV}",
     env={"region": os.environ["CDK_DEFAULT_REGION"], "account": os.environ["CDK_DEFAULT_ACCOUNT"]},
     datasets_table=storage.datasets_table,
+    datalake_users_role=users.datalake_users_role,
```

Comment:
`datalake` seems redundant in both of these places; that should be obvious from context.

Reply:
Yes, I agree, normally I try to avoid it. Fixed now in 8c9bfeb5acd45abcf714e37ae962ca125b4d7d75